### PR TITLE
feat(Illustrations): chain components ready

### PR DESCRIPTION
**Pull request by locothedev**
Lets go wild testing on this one especially on the color part we surely can improve but for now its enough. 

`<Illustration  logo="binance" fillInline="yellow" fillOutline="black" size="m" />`

```
size = "xs" | "s" | "m" | "l" | "xl";
logo = "ethereum" | "binance" | "polygon" | "arbitrum" | "avalanche" | "fantom"
fillInline = can be any color format (Color inside)
fillOutline = can be any color format (Border color)
```

Note: fillInlineLeft and fillInlineRight will be ignored by all but not for ethereum and arbitrum

### CODE REVIEW

File: src/components/Illustrations/types.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+     */
+    fillInlineLeft?: string;
+
+    /**
+     * Use for ethereum and arbitrum only
+     */
+    fillInlineRight?: string
+}
+
+export type size = "xs" | "s" | "m" | "l" | "xl";
```
Comment (BillyG83) [inline src/components/Illustrations/types.ts:31]: 31, 32, 33 please check the types of `LinkTo` Oscar has a cooler way to do this

Comment (locothedev) [reply src/components/Illustrations/types.ts]: done

**Review by BillyG83 — APPROVED** [review]

**Comment by BillyG83** [comment]
fails on `size / size (pull_request) Failing after 3m — size` but i think Dani can use this on his component to fix these errors

**Comment by locothedev** [comment]
Yes thats the plan!

**Pull request merged**